locales/en-US/app.ftl

```diff
@@ -260,3 +260,10 @@ warningMissingPrefTitle=Developing Test Pilot?
 warningMissingPrefDetail=When running Test Pilot locally or in development environments, special configuration is required. Please see <a>our documentation</a> for details.
 warningBadHostnameTitle=Unapproved hostname!
 warningBadHostnameDetail=The Test Pilot site may only be accessed from testpilot.firefox.com, testpilot.stage.mozaws.net, testpilot.dev.mozaws.net, or example.com:8000. Please see <a>our documentation</a> for details.
+
+// Text to show to users when an experiment has graduated
+[[graduated]]
+graduationReportButton=Graduation Report
+graduationNoticeTitle=This experiment has ended
+graduationNoticeReportNotReady=We are working on a full report. Check back soon for the details.
```

Comment:
Why are we duplicating two existing strings? Are those used anywhere?

```
experimentGradReportPendingTitle = This experiment has ended
experimentGradReportPendingCopy = We are working on a full report. Check back soon for the details.
```

Reply:
Thank you for pointing this out! I searched for "graduation" but there were no hits, and "grad" hit a bunch of other things that were not related. I will re-use the existing strings.